src/modules/muorb/krait/px4muorb_KraitRpcWrapper.cpp

```diff
@@ -111,6 +111,9 @@ int calc_timer_diff_to_dsp_us(int32_t *time_diff_us)
 		return -3;
 	}
 
+	// The clock count needs to get converted to us. The clock supposedly
+	// runs at just under 20 MHz. The magic value of 19.2 was provided by
```

Comment:
The "just under 20MHz" probably doesn't need to be mentioned -- that was what I experimentally determined before Qualcomm gave me example code that assumes it's exactly 19.2MHz.